src/Resistor/Resistor.ts

```diff
@@ -0,0 +1,48 @@
+import ResistorColorEntry from './ResistorColorEntry';
+
+class Resistor {
+  // tslint:disable max-line-length
+  //                                                                         Name      Color      Multiplier Value Tolerance
+  public static readonly BLACK: ResistorColorEntry =  new ResistorColorEntry('Black',  '#000000', 1,            0, undefined);

```

Comment:
Should we take the tolerance values as numbers and then just format them on the fly?

Reply:
Sure.  In my head I kept debating whether I should store "5%" as 5 or 0.05, and settled it by going for string, but I can also just rename the variable "toleranceInPercent".